.github/workflows/tests.yml

```diff
@@ -7,17 +7,17 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
-      - name: Use Xcode 15
-        run: sudo xcode-select -switch /Applications/Xcode_15.0.app
+      - name: Use Xcode 14.3
+        run: sudo xcode-select -switch /Applications/Xcode_14.3.app
       - name: Run Unit Tests
-        run: set -o pipefail && xcodebuild -workspace 'BraintreeDropIn.xcworkspace' -sdk 'iphonesimulator' -configuration 'Debug' -scheme 'UnitTests' -destination 'name=iPhone 15,platform=iOS Simulator'  test  | xcpretty
+        run: set -o pipefail && xcodebuild -workspace 'BraintreeDropIn.xcworkspace' -sdk 'iphonesimulator' -configuration 'Debug' -scheme 'UnitTests' -destination 'name=iPhone 14,OS=16.4,platform=iOS Simulator'  test  | xcpretty
   ui_test_job:
     name: UI
     runs-on: macOS-13
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
-      - name: Use Xcode 15
-        run: sudo xcode-select -switch /Applications/Xcode_15.0.app
+      - name: Use Xcode 14.3
+        run: sudo xcode-select -switch /Applications/Xcode_14.3.app
       - name: Run UI Tests
-        run: set -o pipefail && xcodebuild -workspace 'BraintreeDropIn.xcworkspace' -sdk 'iphonesimulator' -configuration 'Debug' -scheme 'UITests' -destination 'name=iPhone 15,platform=iOS Simulator'  test  | xcpretty
+        run: set -o pipefail && xcodebuild -workspace 'BraintreeDropIn.xcworkspace' -sdk 'iphonesimulator' -configuration 'Debug' -scheme 'UITests' -destination 'name=iPhone 14,OS=16.4,platform=iOS Simulator'  test  | xcpretty
```

Comment:
The latest iOS version provided by Xcode is what this defaults to. Isn't 16.4 the latest the Xcode 14.3 offers? See https://xcodereleases.com/

Reply:
It _seems_ like iOS 17 may be the default for the runner but you may be right, it's a bit unclear. Was looking at this issue: https://github.com/actions/runner-images/issues/8023

Reply:
It does seem to have failed faster but not sure if that is accurate. We can also merge main into the 2 latest PRs to see if the Xcode 15 changes help there. 🤷